stable/external-dns/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: |
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.7.10
+version: 1.7.11
```

Comment:
Lets make this a minor version bump as this change adds a new feature to the chart

Reply:
Done.